ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/RpcMethod.java

```diff
@@ -58,7 +58,7 @@ public enum RpcMethod {
   ENGINE_GET_PAYLOAD_BODIES_BY_HASH_V1("engine_getPayloadBodiesByHashV1"),
   ENGINE_GET_PAYLOAD_BODIES_BY_RANGE_V1("engine_getPayloadBodiesByRangeV1"),
   ENGINE_EXCHANGE_CAPABILITIES("engine_exchangeCapabilities"),
-
+  ENGINE_PREPARE_PAYLOAD_DEBUG("engine_preparePayload_debug"),
```

Comment:
Here is a tricky question, shouldn't this method be in the debug set? Another reason to move that is the way we built the engine_exchange_capabilities might report this method. CL is probably going to ignore it though.

Reply:
DEBUG is present even if the ENGINE api is not, so I think this needs to live within engine api.  Good point about the capabilities though 🤔 I will check to see if this causes problems for CLs

Reply:
What about a ENGINE_DEBUG set? So it is separated from ENGINE and can be enabled only if used

Reply:
I filtered the debug prepare payload from the capabilities list in the same way the capabilities list filters itself 👍

Reply:
just saw the ENGINE_DEBUG set suggestion.  I think we could add something like that if we have more than one engine debug endpoint in the future 👍 .  Rule of three applies in this case IMO